README.md

```diff
@@ -503,6 +504,23 @@ List of supported Cloudflare services:
   * `cloudflare_zone`
   * `cloudflare_record`
 
+
+### Use with Logz.io
+
+Example:
+
+```
+ LOGZIO_API_TOKEN=foobar LOGZIO_BASE_URL=https://api-eu.logz.io ./terraformer import logzio -r=alerts,alert_notification_endpoints // Import Logz.io alerts and alert notification endpoints
```

Comment:
You can add flag api-token and base-url. See example in github provider. not must...

Reply:
We tried with this approach first, but it didn't work as the provider also needs the same env vars set. So we removed the flags for those arguments and instead just use the env vars.